_blogposts/2020-09-25-release-8-3-2.mdx

```diff
@@ -0,0 +1,109 @@
+---
+author: hongbo
+date: "2020-09-26" 
+previewImg:
+category: compiler
+title: What's new in ReScript 8.3 (Part 2)
+description: |
+---
+
+## Introduction
+
+ReScript is a soundly typed language with an optimizing compiler focused on the JS platform. 
+It's focused on type safety, performance and JS interop. It used to be called BuckleScript.
+
+[ReScript@8.3](https://www.npmjs.com/package/bs-platform/v/8.3.0) is now available for testing, you can try it via
+
+```
+npm i bs-platform@8.3.0
+```
+
+Following the [previous post](https://rescript-lang.org/blog/release-8-3-pt1), in this post we will go through 
```

Comment:
When you are referencing other blog posts on our website, you can omit the `https://rescript-lang.org` part. The non-absolute form is preferred, since it allows us to verify all hour link hrefs inside each document.

```suggestion
Following the [previous post](/blog/release-8-3-pt1), in this post we will go through 
```